static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -131,17 +134,22 @@ define([
     }
 
     function ContributionsABTestVariant(options, test) {
+        var trackingCampaignId = test.epic ? 'epic_' + test.campaignId : test.campaignId;
+
         this.campaignId = test.campaignId;
         this.id = options.id;
         this.maxViews = options.maxViews || maxViews;
         this.isUnlimited = options.isUnlimited || false;
 
-        this.contributeURL = options.contributeURL || this.makeURL(contributionsURL, test.contributionsCampaignPrefix);
-        this.membershipURL = options.membershipURL || this.makeURL(membershipURL, test.membershipCampaignPrefix);
+        this.pageviewId = (config.ophan && config.ophan.pageViewId) || 'not_found';
```

Comment:
It is used here : https://github.com/guardian/frontend/pull/15966/files#diff-22ef0d3c397978fc2d0f8b33d31fa204R200